<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="JI-STD-95YR373" failures="1" tests="6" name="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" time="95.428" errors="0" timestamp="2021-01-26T23:07:56 IST" skipped="3">
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validateadminloginpage" time="33.514"/>
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validateanalystloginpage" time="30.033"/>
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validateleadanalyst" time="31.881">
    <failure message="Invalid password" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Invalid password
at org.testng.Assert.fail(Assert.java:97)
at com.JFMcom.qa.PageTest.ATC_0001_LoginTest.validateleadanalyst(ATC_0001_LoginTest.java:125)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateleadanalyst -->
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validateSMEloginpage">
    <skipped/>
  </testcase> <!-- validateSMEloginpage -->
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validatereviewerloginpage">
    <skipped/>
  </testcase> <!-- validatereviewerloginpage -->
  <testcase classname="com.JFMcom.qa.PageTest.ATC_0001_LoginTest" name="validateSMEandAnalyst">
    <skipped/>
  </testcase> <!-- validateSMEandAnalyst -->
</testsuite> <!-- com.JFMcom.qa.PageTest.ATC_0001_LoginTest -->
